float_data_for_tock.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+import json
+
+api_key = os.environ.get('FLOAT_API_KEY')
+headers = {'Authorization': 'Bearer ' + api_key}
+endpoint = 'tasks'
+year = "2016"
+month = "04"
+day = "01"
+weeks = "52"
```

Comment:
We could set this up so the script or command takes in input for these but also provide sensible defaults (such as calculating and using the current Fiscal Year).